package pro.kensait.java.exception;

public class ExceptionMain1 {

    public static void main(String[] args) {
        int value1 = 10, value2 = 0, answer = 0;
        answer = value1 / value2;
        System.out.println("answer ---> " + answer);
    }
}